todoman/model.py

```diff
@@ -91,9 +93,12 @@ def __init__(self, todo=None, filename=None, mtime=None, new=False):
 
         if new:
             now = datetime.now(self._localtimezone)
+            config = load_config()
```

Comment:
Also this breaks tests where the `Todo` class is tested without a config environment around it.